### ISSUE TITLE
Editor crash in Variant::stringify

### ISSUE THREAD

**Issue by qarmin** [godotengine/godot#32572]
**Godot version:**
3.2.dev.custom_build. 6d6d43714
**OS/device including version:**
Ubuntu 19.04
**Issue description:**
Backtrace
```
[1] /lib/x86_64-linux-gnu/libc.so.6(+0x43f60) [0x7fb1b1dd7f60] (??:0)
[2] Object::to_string() (/home/rafal/Pulpit/mojgodot/core/object.cpp:941)
[3] Variant::stringify(List<void const*, DefaultAllocator>&) const (/home/rafal/Pulpit/mojgodot/core/variant.cpp:1584)
[4] Variant::operator String() const (/home/rafal/Pulpit/mojgodot/core/variant.cpp:1400)
[5] GDScriptFunctions::call(GDScriptFunctions::Function, Variant const**, int, Variant&, Variant::CallError&) (/home/rafal/Pulpit/mojgodot/modules/gdscript/gdscript_functions.cpp:681)
[6] GDScriptFunction::call(GDScriptInstance*, Variant const**, int, Variant::CallError&, GDScriptFunction::CallState*) (/home/rafal/Pulpit/mojgodot/modules/gdscript/gdscript_function.cpp:1148)
[7] GDScriptInstance::call_multilevel(StringName const&, Variant const**, int) (/home/rafal/Pulpit/mojgodot/modules/gdscript/gdscript.cpp:1180)
[8] Node::_notification(int) (/home/rafal/Pulpit/mojgodot/scene/main/node.cpp:58)
[9] Node::_notificationv(int, bool) (/home/rafal/Pulpit/mojgodot/./scene/main/node.h:46 (discriminator 14))
[10] CanvasItem::_notificationv(int, bool) (/home/rafal/Pulpit/mojgodot/./scene/2d/canvas_item.h:166 (discriminator 3))
[11] Control::_notificationv(int, bool) (/home/rafal/Pulpit/mojgodot/./scene/gui/control.h:48 (discriminator 3))
[12] Container::_notificationv(int, bool) (/home/rafal/Pulpit/mojgodot/./scene/gui/container.h:38 (discriminator 3))
[13] TabContainer::_notificationv(int, bool) (/home/rafal/Pulpit/mojgodot/scene/gui/tab_container.h:38 (discriminator 3))
[14] Object::notification(int, bool) (/home/rafal/Pulpit/mojgodot/core/object.cpp:933)
[15] SceneTree::_notify_group_pause(StringName const&, int) (/home/rafal/Pulpit/mojgodot/scene/main/scene_tree.cpp:957)
[16] SceneTree::idle(float) (/home/rafal/Pulpit/mojgodot/scene/main/scene_tree.cpp:515 (discriminator 3))
[17] Main::iteration() (/home/rafal/Pulpit/mojgodot/main/main.cpp:1917)
[18] OS_X11::run() (/home/rafal/Pulpit/mojgodot/platform/x11/os_x11.cpp:3181)
[19] godot(main+0x128) [0x139f60a] (/home/rafal/Pulpit/mojgodot/platform/x11/godot_x11.cpp:57)
[20] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xeb) [0x7fb1b1dbab6b] (??:0)
[21] godot(_start+0x2a) [0x139f42a] (??:?)
```
**Steps to reproduce:**
Run project or ControlALL.tscn scene and wait
**Minimal reproduction project:**
[The_worst_Godot_test_project.zip](https://github.com/godotengine/godot/files/3693604/The_worst_Godot_test_project.zip)

**Comment by Anutrix** [comment]
Couldn't reproduce on latest master https://github.com/godotengine/godot/commit/1d9233c3882afe888b9396f7f2aac917d4dcac4d with given project in 10 mins.

But the Windows clipboard was cr*zy. Run project and try copy pasting anywhere outside Godot(like Firefox, Notepad++, etc.). Might be unrelated

**Comment by qarmin** [comment]
Better project, should crash a little after start - [The_worst_Godot_test_project.zip](https://github.com/godotengine/godot/files/3694590/The_worst_Godot_test_project.zip)

Errors are expected, because project [The Worst Godot Test Project](https://github.com/qarmin/The-worst-Godot-test-project) aims to test every function with the strangest set of parameters(too big, too small etc.)

**Comment by qarmin** [comment]
This is smallest reproduction script
TabContainer Node:
```
extends TabContainer

func _process(delta) -> void:
	var qq : String = ""
	qq = qq
	
	var popup_menu_temp : PopupMenu = PopupMenu.new()
	if randi() % 2 == 1:
		set_popup(popup_menu_temp)
	if randi() % 2 == 1:
		qq += str(get_popup())
	popup_menu_temp.queue_free()
```

**Comment by qarmin** [comment]
Fixed by #39806
